stakingModel/test/GoodFundManager.test.js

```diff
@@ -125,10 +127,12 @@ contract("GoodFundManager - transfer interest from the staking contract to the r
     let cdaiGains = gains["0"];
     let reserveCDaiBalanceBefore = await cDAI.balanceOf(goodReserve.address);
     let tx = await goodFundManager.transferInterest(simpleStaking.address);
+    let homeAvatarAfter = await goodDollar.balanceOf(homeAvatar);
     let reserveCDaiBalanceAfter = await cDAI.balanceOf(goodReserve.address);
     let stakingGDBalance = await goodDollar.balanceOf(simpleStaking.address);
     const gdPriceAfter = await marketMaker.currentPrice(cDAI.address);
-    expect(stakingGDBalance.toString()).to.be.equal("970492");
+    expect(stakingGDBalance.toString()).to.be.equal("0");
+    expect(homeAvatarAfter.toString()).to.be.equal("971086");
```

Comment:
hard coded numbers need to be explained
also why did the amount change from 970492 to 971086?

Reply:
updated.
906867c66fb34ed7de644ef09ce6699fd747c0a8 

changes in reserve contract:
gdInterestMinted: 970492 (type: uint256),
gdExpansionMinted: 594 (type: uint256)